console/src/channel/services/ontology.tsx

```diff
@@ -241,12 +270,23 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props) => {
         name: resources[0].name,
         ontologyID: resources[0].id.payload,
       }),
+    openCalculated: () => openCalculated(props),
   };
   const singleResource = resources.length === 1;
+  const isCalculated = singleResource && resources[0].data?.expression !== "";
```

Comment:
Isn't there an `isCalculated` function inside of the typescript client? Should use that instead